index.html

```diff
@@ -39,7 +40,166 @@ <h2 id="description-text">I'm a Software Developer</h2>
                          
                 </div>
             </div>  
-        </section>     
-    </main>
+        </section> 
+        <section id="work-section">
+                <section class="recent-work">
+                    <h2 id="recent-work-title">My Recent Works</h2>
+                    <hr>
+                    <img id="img-project" src="images/Img-Placeholder.png" alt="images-work">
+                    <h3 id="project-description-title">Multi-Post Stories</h3>
+                    <p id="project-description">A daily selection of privately personalized reads;
+                       no accounts or sign-ups required. has been the industry's standard dummy
+                       text ever since the 1500s, when an unknown printer took a standard dummy text.
+                    </p>
```

Comment:
Your Design
![image](https://user-images.githubusercontent.com/78251927/227324095-d6bada8f-e50a-4d5d-8983-04ff0e1396e8.png)

Figma
![image](https://user-images.githubusercontent.com/78251927/227324296-2d17f7e3-a575-4f42-a005-384a3b6e7dda.png)

Please kindly fix the issue with the `See Project` button so that it displays properly as well as the broken image. 💪